recipes/fswatch/all/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.5)
+project(cmake_wrapper)
+
+message(STATUS "Conan CMake Wrapper")
+add_subdirectory("src")
```

Comment:
Why is this needed? This is doing nothing, and directly using upstream's CMakeLists would work

Reply:
Thanks! I removed the wrapper